src/cc2olx/tools/README.rst

```diff
@@ -0,0 +1,61 @@
+Tools
```

Comment:
Can you link the read me here into the overall repo's readme so others can see these instruction from the repo's home page?

Reply:
Yes, I will do that.